tests/02-compile-arm-ports/Makefile

```diff
@@ -59,6 +67,10 @@ platform-specific/cc26x0-cc13x0/cc26x0-web-demo/cc26x0-cc13x0:BOARD=sensortag/cc
 platform-specific/cc26x0-cc13x0/base-demo/cc26x0-cc13x0:BOARD=sensortag/cc2650 \
 platform-specific/cc26x0-cc13x0/base-demo/simplelink:BOARD=sensortag/cc1350 \
 platform-specific/cc26x0-cc13x0/base-demo/simplelink:BOARD=sensortag/cc1352r1 \
+platform-specific/nrf/empty-application/nrf:BOARD=nrf52840/dk \
```

Comment:
What's the idea behind testing this for the nRF52840?

Reply:
The idea behind is to make sure it's skipped in the Makefile and if it is not, it should fail the unittest.
Maybe it's not needed, I can remove if it is too much.

Reply:
Makes sense, that's a good test to have.